aws/cluster/networking.tf

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
This file HAS to be removed, it was removed on purpose because it creates VPC while we use existing VPC (see my other comment about handling network layer differently)

Reply:
will do. Good that I did not use the var formatting using it :)